app/controllers/users_controller.rb

```diff
@@ -110,15 +110,16 @@ def onboarding_update
 
   def join_org
     authorize User
-    if (@organization = Organization.find_by(secret: params[:org_secret]))
+    if (@organization = Organization.find_by(secret: "params[:org_secret].strip"))
```

Comment:
I think the quotes here should be removed

Reply:
Ooh yes, good catch. I was debugging 😅 
```suggestion
    if (@organization = Organization.find_by(secret: params[:org_secret].strip))
```